pkg/codegen/testing/test/program_driver.go

```diff
@@ -74,6 +74,12 @@ var PulumiPulumiProgramTests = []ProgramTest{
 		// Blocked on nodejs: TODO[pulumi/pulumi#8068]
 		// Flaky in go: TODO[pulumi/pulumi#8123]
 	},
+	{
+		Directory:   "aws-iam-policy",
+		Description: "AWS IAM Policy",
+		SkipCompile: codegen.NewStringSet("go"),
+		// Blocked on go
```

Comment:
I found this issue while looking into #10033 - I'm still working on that one, and once it's resolved, this can be removed.

Reply:
Looking at this more closely, I think we can actually _not_ skip `go` here.  The issue in #10033 appears to actually be related to the resulting pcl generated by `pulumi-yaml` rather than the pcl -> go codegen process.